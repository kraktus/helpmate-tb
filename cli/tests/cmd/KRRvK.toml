bin.name = "helpmate_tb_cli"
args = "--no-time generate -m \"KRRvK\" --tb-dir ../table --recursive"
