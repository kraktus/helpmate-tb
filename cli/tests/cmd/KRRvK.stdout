[INFO  helpmate_tb_cli::generate] Building KRvK with winner: white
[DEBUG helpmate_tb::generation] all_pos_vec capacity before generating: 35154
[DEBUG helpmate_tb::generation] all_pos_vec capacity after generating: 35154
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 29560 after shrinking
[DEBUG helpmate_tb::generation] nb pos 29560
[DEBUG helpmate_tb::generation] counter 62664
[DEBUG helpmate_tb::generation] nb White mate 27
[DEBUG helpmate_tb::generation] nb Black mates 0
[DEBUG helpmate_tb_cli::explore] From White perspective, win: 49952, draw: 63, lost: 0, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 84%
[DEBUG helpmate_tb_cli::explore] Win(0), 27
[DEBUG helpmate_tb_cli::explore] Win(1), 189
[DEBUG helpmate_tb_cli::explore] Win(2), 294
[DEBUG helpmate_tb_cli::explore] Win(3), 1372
[DEBUG helpmate_tb_cli::explore] Win(4), 3270
[DEBUG helpmate_tb_cli::explore] Win(5), 5248
[DEBUG helpmate_tb_cli::explore] Win(6), 9039
[DEBUG helpmate_tb_cli::explore] Win(7), 7893
[DEBUG helpmate_tb_cli::explore] Win(8), 10129
[DEBUG helpmate_tb_cli::explore] Win(9), 6181
[DEBUG helpmate_tb_cli::explore] Win(10), 4473
[DEBUG helpmate_tb_cli::explore] Win(11), 1041
[DEBUG helpmate_tb_cli::explore] Win(12), 734
[DEBUG helpmate_tb_cli::explore] Win(13), 35
[DEBUG helpmate_tb_cli::explore] Win(14), 27
[INFO  helpmate_tb_cli::generate] Building KRvK with winner: black
[DEBUG helpmate_tb::generation] all_pos_vec capacity before generating: 35154
[DEBUG helpmate_tb::generation] all_pos_vec capacity after generating: 35154
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 29560 after shrinking
[DEBUG helpmate_tb::generation] nb pos 29560
[DEBUG helpmate_tb::generation] counter 62664
[DEBUG helpmate_tb::generation] nb Black stalemate/capture resulting in draw 9
[DEBUG helpmate_tb::generation] nb White mates 27
[DEBUG helpmate_tb_cli::explore] From Black perspective, win: 0, draw: 49988, lost: 27, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 84%
[DEBUG helpmate_tb_cli::explore] Lose(0), 27
[INFO  helpmate_tb_cli::generate] Building KRRvK with winner: white
[DEBUG helpmate_tb::generation] all_pos_vec capacity before generating: 2144394
[DEBUG helpmate_tb::generation] all_pos_vec capacity after generating: 2144394
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 1891839 after shrinking
[DEBUG helpmate_tb::generation] nb pos 1891839
[DEBUG helpmate_tb::generation] counter 1959076
[DEBUG helpmate_tb::generation] nb White mate 9052
[DEBUG helpmate_tb::generation] nb Black mates 0
[DEBUG helpmate_tb_cli::explore] From White perspective, win: 1372472, draw: 2468, lost: 0, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 36%
[DEBUG helpmate_tb_cli::explore] Win(0), 9052
[DEBUG helpmate_tb_cli::explore] Win(1), 41612
[DEBUG helpmate_tb_cli::explore] Win(2), 112954
[DEBUG helpmate_tb_cli::explore] Win(3), 310128
[DEBUG helpmate_tb_cli::explore] Win(4), 598049
[DEBUG helpmate_tb_cli::explore] Win(5), 163997
[DEBUG helpmate_tb_cli::explore] Win(6), 123987
[DEBUG helpmate_tb_cli::explore] Win(7), 4964
[DEBUG helpmate_tb_cli::explore] Win(8), 4863
[DEBUG helpmate_tb_cli::explore] Win(9), 1
[DEBUG helpmate_tb_cli::explore] Win(10), 2367
[DEBUG helpmate_tb_cli::explore] Win(12), 475
[DEBUG helpmate_tb_cli::explore] Win(14), 23
[INFO  helpmate_tb_cli::generate] Building KRRvK with winner: black
[DEBUG helpmate_tb::generation] all_pos_vec capacity before generating: 2144394
[DEBUG helpmate_tb::generation] all_pos_vec capacity after generating: 2144394
[DEBUG helpmate_tb::generation] all_pos_vec capacity: 1891839 after shrinking
[DEBUG helpmate_tb::generation] nb pos 1891839
[DEBUG helpmate_tb::generation] counter 1959076
[DEBUG helpmate_tb::generation] nb Black stalemate/capture resulting in draw 2468
[DEBUG helpmate_tb::generation] nb White mates 9052
[DEBUG helpmate_tb_cli::explore] From Black perspective, win: 0, draw: 1365888, lost: 9052, unkown: 0
[DEBUG helpmate_tb_cli::explore] Index density = 36%
[DEBUG helpmate_tb_cli::explore] Lose(0), 9052
